src/android/InAppBrowser.java

```diff
@@ -534,6 +534,16 @@ public void onPageFinished(WebView view, String url) {
                             dialog.dismiss();
                             dialog = null;
                         }
+                        
+                        // Fix for webView window not being destroyed correctly causing memory leak
+                        // (https://github.com/apache/cordova-plugin-inappbrowser/issues/290)
+                        if (url.equals(new String("about:blank"))) {
```

Comment:
The call to `new String()` is unnecessary, consider doing: `if (url.equals("about:blank")) {`